webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import URI from 'urijs';
+import { Spinner } from 'patternfly-react';
+import Table from '../common/table';
+import { STATUS } from '../../constants';
+import './ModelsTable.css';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.updateSearch(this.props.data);
+  }
+  render() {
+    const {
+      schema,
+      perPage,
+      page,
+      searchQuery,
+      results,
+      sortBy,
+      sortOrder,
+    } = this.props;
+    return this.props.status === STATUS.PENDING ? (
```

Comment:
Wouldn't it make more sense to pass a boolean `loading` prop and move the `this.props.status === STATUS.PENDING` logic to a redux selector and use it in `mapStateToProps`?

Reply:
IMO I prefer it as well:

```js
if (loading) {
  return <LoadingTable />;
}

return <Table ... />;
```